low-level/sensors/impl/generic/node_hw_alerts_info.py

```diff
@@ -47,5 +47,91 @@
+        **dict.fromkeys(["Presence detected", "Presence detected ()"], {
+            "Asserted": Alert(
+                "insertion", "informational",
+                "'Power Supply {}' is present. [{}]",
+                # Intel/Supermicro: 'Power Supply 1' is present. [PS1 (0x85)]
+                "Power Supply Port is being monitored.", "None"),
+            "Deasserted": Alert(
+                "missing", "critical",
+                "'Power Supply {}', is missing. [{}]",
+                "Power supply redundancy is affected, if more power " +
+                "supplies go down or missing, server may go offline.",
+                "Install missing Power supply immediately.")
+        }),
+        **dict.fromkeys(["Predictive failure", "Predictive failure ()"], {
+            "Asserted": Alert(
+                "fault", "warning",
+                "Failure is predicted for 'Power Supply {}'. [{}]",
+                "Power Supply Redundancy will be affected. If more power " +
+                "supplies go down or missing, server may go offline.",
+                "Please replace the faulty Power supply soon."
+            ),
+            "Deasserted": Alert(
+                "fault_resolved", "informational",
+                "No Failure threat anymore for 'Power Supply {}'. [{}]",
+                "Power Supply Redundancy is in good state.", "None"
+            )
+        }),
+        **dict.fromkeys(["Failure detected", "Failure detected ()"], {
+            "Asserted": Alert(
+                "fault", "error",
+                "Failure detected for 'Power Supply {}'. [{}]",
+                "Power Supply Redundancy is affected. If more power " +
+                "supplies go down or missing, server may go offline.",
+                "Please replace the faulty Power Supply immediately."
+            ),
+            "Deasserted": Alert(
+                "fault_resolved", "informational",
+                "'Power Supply {}' recovered from the failure. [{}]",
+                "Power Supply Redundancy is in good state.", "None"
+            )
+        }),
+        **dict.fromkeys(["Config Error", "Config Error ()"], {
+            "Asserted": Alert(
+                "fault", "error",
+                "'Power Supply {}' has configuration error. [{}]",
+                "Power Supply Redundancy is affected. If error is not fixed, "+
+                "and other power supplies go down, server may go offline.",
+                DEFAULT_RECOMMENDATION
+            ),
+            "Deasserted": Alert(
+                "fault_resolved", "informational",
+                "Configuration error resolved for 'Power Supply {}'. [{}]",
+                "Power Supply Redundancy is in good state.", "None"
+            )
+        }),
+        **dict.fromkeys(["Power Supply AC lost", "Power Supply AC lost ()"], {
+            "Asserted": Alert(
+                "fault", "critical",
+                "'Power Supply {}' lost the AC supply. [{}]",
```

Comment:
Recommendation -> Check power supply cable connectivity. If power cable also seen connected, cable may be faulty, check & replace it.